PF/PF-Assignment-1/Palindrome using Stack/StackPalindrome.java

```diff
@@ -0,0 +1,38 @@
+import java.util.Stack;
+import java.util.Scanner;
+
+/**
+ * This Class checks whether a String is Palindrome or not using Stack
```

Comment:
In this question, you need to use stack that is implemented in previous question.